tests/installation/partitioning_olddisk.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary:  Install on old disk
```

Comment:
Sorry I don't understand what this means. Could you please rephrase it with a bit more words?

Reply:
Updated.